x-pack/plugins/fleet/server/services/epm/packages/install.ts

```diff
@@ -1206,17 +1213,65 @@ export async function restartInstallation(options: {
   await savedObjectsClient.update(PACKAGES_SAVED_OBJECT_TYPE, pkgName, savedObjectUpdate);
 }
 
+function loadYmlFromAssetMap(assetsMap: AssetsMap, path: string) {
+  if (!assetsMap.has(path)) {
+    return undefined;
+  }
+  return yaml.load(assetsMap.get(path)!.toString('utf-8'));
+}
+
+/**
+ * Returns a map of the data stream path fields to elasticsearch index pattern.
+ * @param dataStreams an array of RegistryDataStream objects
+ */
+function generateESIndexPatterns(
+  dataStreams: RegistryDataStream[] | undefined,
+  packageInfo: { name: string; version: string },
+  assetsMap: AssetsMap
+): EsIndexPattern[] {
+  if (!dataStreams) {
+    return [];
+  }
+
+  return dataStreams
+    .map((dataStream) => {
+      return {
+        title: dataStream.path,
+        name: generateTemplateIndexPattern(dataStream),
```

Comment:
it might not be needed to load the title from the manifest.yml, I noticed the dataStream.title contains the same text (tested with apache):
```
 .map((dataStream) => {
      return {
        title: dataStream.path,
        name: generateTemplateIndexPattern(dataStream),
        display_name: dataStream.title,
      };
    })
```

It creates

```
 [
  {
    "title": "access",
    "name": "logs-apache.access-*",
    "display_name": "Apache access logs"
  },
  {
    "title": "error",
    "name": "logs-apache.error-*",
    "display_name": "Apache error logs"
  },
  {
    "title": "status",
    "name": "metrics-apache.status-*",
    "display_name": "Apache status metrics"
  }
]
```